check_test.go

```diff
@@ -122,6 +122,9 @@ func TestTimeComparison(t *testing.T) {
 	if err := os.Chtimes(fh.Name(), epoch, epoch); err != nil {
 		t.Fatal(err)
 	}
+	if err := os.Chtimes(dir, epoch, epoch); err != nil {
```

Comment:
Notice that if we removed these lines, this test _should_ have failed because "." wouldn't have been created at `time=5.000000000` (it would have been created at a time like 1468935093.642931635). But this test doesn't, unless we make the change above with `creator.curSet`.